### PR TITLE
Replace old reference architecture image with a new one we have the source for

### PR DESCRIPTION
**Pull request by andrew-dawson-seqera**
We lost the original source of the Seqera reference architecture PNG that's under the Enterprise installation section of the public docs. I've recreated a draw.io source file and written that source as a reference here.

### CODE REVIEW

File: platform_versioned_docs/version-24.2/enterprise/_images/old_source_seqera_reference_architecture.png
Comment (ewels) [inline platform_versioned_docs/version-24.2/enterprise/_images/old_source_seqera_reference_architecture.png:1]: Do we need to keep the old one? I'd generally prefer to delete, git history keeps the old one for us :)

Comment (pditommaso) [reply platform_versioned_docs/version-24.2/enterprise/_images/old_source_seqera_reference_architecture.png:1]: Also the "new" still does not include Wave, Fusion. It should be updated (also removing "Angular", that's irrelevant)

Comment (andrew-dawson-seqera) [reply platform_versioned_docs/version-24.2/enterprise/_images/old_source_seqera_reference_architecture.png:1]: With these last two commits, I've addressed the following:

- [x] Deleted the old diagram, @ewels was right that there is no need to keep it.

Uploaded a new diagram with the following changes, per @pditommaso feedback.
- [x] Removed Angular references.
- [x] Renamed the Compute platform(s) section to "Compute environment(s)" to match its name in the Web UI, API, etc.
- [x] Added in Wave and Fusion references to the Compute environment(s) section.
- [x] Added in Data Studios under the User access section.
- [x] Collapsed redundant references in the Kubernetes subsection of the Supported computing platforms section.

**Comment by llewellyn-sl** [comment]
@andrew-dawson-seqera Thanks for putting this diagram together! 

Two things:
- Do we need the word "same" for the Nextflow, Wave, and Fusion subtext here? 
image

- The image compression makes it hard to read the subtext in some sections (though opening the full image in a separate tab is perfect). @jason-seqera would the [ideal-image plugin](https://docusaurus.io/pt-BR/docs/api/plugins/@docusaurus/plugin-ideal-image) help us to render these images in higher quality by default?

File: platform_versioned_docs/version-24.2/enterprise/_images/old_source_seqera_reference_architecture.png
Comment (pditommaso) [reply platform_versioned_docs/version-24.2/enterprise/_images/old_source_seqera_reference_architecture.png:1]: Wave is co-located with Platform, therefore it should the "Application server" box

**Comment by andrew-dawson-seqera** [comment]
>Do we need the word "same" for the Nextflow, Wave, and Fusion subtext here? I would opt for just "the compute environment", unless there's nuance here that I'm missing?

@llewellyn-sl - good point on the wording. I was carrying forward the language that was there before. I think the original writer was trying to specify that all of the activity in that box is happening on _one_ compute environment (and not split across multiple). That nuance is something we can clarify in the supporting documentation, however, and doesn't add anything but confusion to the diagram. I've adopted your suggested changes in the latest commit.

>The image compression makes it hard to read the subtext in some sections (though opening the full image in a separate tab is perfect). @jason-seqera would the [ideal-image plugin](https://docusaurus.io/pt-BR/docs/api/plugins/@docusaurus/plugin-ideal-image) help us to render these images in higher quality by default?

I'm glad you brought up rendering. I was worried it wouldn't come out cleanly, I'm not very knowledgeable at that kind of thing. @jason-seqera - I welcome suggestions and/or direct edits to the source file to make it render more cleanly. 🙏

**Review by jason-seqera — APPROVED** [review]

**Pull request merged**